src/ui/server/controllers/insurance/policy-and-export/single-contract-policy/map-submitted-data/index.ts

```diff
@@ -0,0 +1,43 @@
+import { RequestBody } from '../../../../../../types';
+import { FIELD_IDS } from '../../../../../constants';
+import createTimestampFromNumbers from '../../../../../helpers/date/create-timestamp-from-numbers';
+
+const {
+  POLICY_AND_EXPORTS: {
+    CONTRACT_POLICY: { REQUESTED_START_DATE },
+  },
+} = FIELD_IDS.INSURANCE;
+
+/**
+ * mapSubmittedData
+ * Check form data and map any fields that need to be sent to the API/DB in a different format or structure.
+ * @param {Express.Request.body} Form data
+ * @returns {Object} Page variables
+ */
+const mapSubmittedData = (formBody: RequestBody): object => {
+  const requestedStartDateFormIds = {
+    dayId: `${REQUESTED_START_DATE}-day`,
+    monthId: `${REQUESTED_START_DATE}-month`,
+    yearId: `${REQUESTED_START_DATE}-year`,
+  };
+
+  const { dayId, monthId, yearId } = requestedStartDateFormIds;
+
+  if (formBody) {
```

Comment:
could we do all the ifs on one line? might look nicer as compared to nested ifs?

Reply:
I did think of this but we will only have to move it back to this in the next ticket as we'll need to add some more logic here